DancingGoat/Views/About/Index.cshtml

```diff
@@ -6,4 +6,4 @@
 
 @Html.DisplayForModel()
 
-<edit-panel content-item-id="@Model.ItemId" language="@Model.Language" />
+@* <edit-panel content-item-id="@Model.ItemId" language="@Model.Language" /> *@
```

Comment:
Why is this commented?

Reply:
Discussed this with @jankalfus (had the same question actually 😆). It is "commented out on purpose" and this is his way of saying so. Should I add a comment or something?